eq-author/package.json

```diff
@@ -111,6 +111,7 @@
     "graphql-tag": "latest",
     "graphql-tools": "latest",
     "jsonwebtoken": "latest",
+    "levenary": "^1.1.1",
```

Comment:
After everything broke on Friday were you still unable to get rid of this?

Reply:
I'll try it again.

Reply:
It's worked just now, it should be fixed in the latest commit. Thank you!